src/components/File/index.tsx

```diff
@@ -1,20 +1,31 @@
-import React from 'react'
-import { FileType, IBaseProps } from '../../models'
+import React, { ReactNode } from 'react'
+import { FileType, IBaseProps, Transference } from '../../models'
+import Icon from '../Icon'
 import Styled from './styles'
 
 interface IProps extends IBaseProps {
   type: FileType
   name: string
+  transference?: Transference
 }
 
 const File: React.FC<IProps> = props => {
-  const { name, isHidden, className } = props
+  const { name, isHidden, className, type, transference } = props
+  const transferenceIcon = (): ReactNode => {
```

Comment:
new line between declarative const and function

Reply:
Done